playroom/snippets.tsx

```diff
@@ -1223,6 +1223,30 @@ const layoutSnippets: Array<Snippet> = [
         />
         </Box>`,
     },
+    {
+        group: 'Mosaic',
+        name: 'HorizontalMosaic',
+        code: `
+        <ResponsiveLayout>
+          <HorizontalMosaic
+            items={Array.from({ length: 6 }, (_, index) => (
+              <SnapCard key={index} title={\`Card \${index + 1}\`} />
```

Comment:
Can you use `<Image />` in snippet instead of snap card? Image components needs width="100%" and height="100%" and this is difficult to know for any designer that wants to use playroom. 

However, with snap card you can replace image with a simple snapcard and mosaic works perfectly